CMakeLists.txt

```diff
@@ -58,7 +58,11 @@ FIND_PACKAGE(Vulkan REQUIRED)
 
 INCLUDE(GNUInstallDirs)
 
-FILE(GLOB VKH_SRC src/*.c src/deps/*.c)
```

Comment:
The cmakelist.txt file has no change needeed.

Reply:
Oops, forgot about the cmake, got it fixed.